### PR TITLE
chore(deps): Bump @babel/* 7.17 -> 7.18

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Follow up to https://github.com/getsentry/sentry/pull/35277

### CODE REVIEW

**Review by priscilawebdev — APPROVED** [review]

**Comment by evanpurkhiser** [comment]
something is broken for the chartcuterie build :/

**Comment by evanpurkhiser** [comment]
nah the error is real

```
Step #1 - "builder-run": b"yarn run v1.22.5
  $ NODE_ENV=production yarn webpack --config=config/webpack.chartcuterie.config.ts --bail
  $ /workspace/node_modules/.bin/webpack --config=config/webpack.chartcuterie.config.ts --bail
  assets by status 1.59 KiB [cached] 1 asset
  ./app/chartcuterie/config.tsx 39 bytes [built] [code generated] [1 error]
  
  ERROR in ./app/chartcuterie/config.tsx
  Module build failed (from ../node_modules/babel-loader/lib/index.js):
  Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@babel/plugin-proposal-class-properties' imported from /workspace/babel-virtual-resolve-base.js
  at new NodeError (/workspace/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:2669:5)
  at packageResolve (/workspace/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3325:9)
  at moduleResolve (/workspace/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3359:18)
  at defaultResolve (/workspace/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3398:13)
  at /workspace/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3421:14
  at Generator.next (<anonymous>)
  at asyncGeneratorStep (/workspace/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:63:103)
  at _next (/workspace/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:65:194)
  at /workspace/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:65:364
  at new Promise (<anonymous>)
  
  webpack 5.72.1 compiled with 1 error in 571 ms
  error Command failed with exit code 1.
  info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
  error Command failed with exit code 1.
  info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
  "

```

**Review by scttcper — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -437,6 +470,15 @@
     "@babel/helper-skip-transparent-expression-wrappers" "^7.16.0"
     "@babel/plugin-proposal-optional-chaining" "^7.16.7"
 
+"@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@^7.17.12":
```
Comment (scttcper) [inline yarn.lock:473]: you've got two versions of things, need to run deduplicate

Comment (evanpurkhiser) [reply yarn.lock:424]: You're right. fixed

**Pull request merged**